tensorflow_serving/model_servers/main.cc

```diff
@@ -191,10 +200,17 @@ int main(int argc, char** argv) {
   tensorflow::string model_name = "default";
   tensorflow::int32 file_system_poll_wait_seconds = 1;
   tensorflow::string model_base_path;
+  tensorflow::int32 model_version_policy = FileSystemStoragePathSourceConfig::LATEST_VERSION;
   std::vector<tensorflow::Flag> flag_list = {
       tensorflow::Flag("port", &port, "port to listen on"),
       tensorflow::Flag("enable_batching", &enable_batching, "enable batching"),
       tensorflow::Flag("model_name", &model_name, "name of model"),
+      tensorflow::Flag("model_version_policy", &model_version_policy,
```

Comment:
@chrisolston  updated to use string and defined the field as VersionPolicy in model_server_config proto. Are you able to review again? thanks